### PR TITLE
main/iptables: upgrade to 1.8.3

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/iptables/APKBUILD [hunk 1]
```diff
@@ -21,16 +21,11 @@ source="https://www.netfilter.org/projects/iptables/files/iptables-$pkgver.tar.b
 builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
-	default_prepare
```
Comment (tcely) [inline main/iptables/APKBUILD]: Why would you remove `default_prepare`??

Comment (J0WI) [reply main/iptables/APKBUILD]: fixed

**Comment by J0WI** [comment]
Patches have been applied upstream.

**Comment by J0WI** [comment]
Done.

**Comment by J0WI** [comment]
fc103a1cc69831ea028c1191e7044f6f9371554a

**Pull request closed without merging**